{
  "candidate": "CVE-2020-14151",
  "description": "** REJECT ** DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: CVE-2018-11813. Reason: This candidate is a duplicate of CVE-2018-11813. Notes: All CVE users should reference [ID] instead of this candidate. All references and descriptions in this candidate have been removed to prevent accidental usage.",
  "notes": {
    "mdeslaur": "appears to be a duplicate of CVE-2018-11813"
  },
  "priority": "low",
  "cvss": {
    "nvd": {
      "vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:N/A:H",
      "score": 7.1,
      "severity": "HIGH"
    }
  },
  "bugs": [
    "https://bugs.gentoo.org/727908"
  ],
  "references": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-14151",
    "http://www.ijg.org/files/jpegsrc.v9d.tar.gz"
  ],
  "public_date": "2020-06-15T17:15:00Z",
  "packages": [
    {
      "name": "libjpeg-turbo",
      "status": "not-affected",
      "note": "2.0.3-0ubuntu1.20.04.1",
      "patches": [
        {
          "source": "upstream",
          "text": "https://github.com/libjpeg-turbo/libjpeg-turbo/commit/909a8cfc7bca9b2e6707425bdb74da997e8fa499"
        }
      ]
    },
    {
      "name": "libjpeg9",
      "status": "not-affected",
      "note": "1:9d-1"
    },
    {
      "name": "libjpeg6b",
      "status": "needs-triage"
    }
  ]
}
